labs/computing-experiment-subjects/index.md

```diff
@@ -0,0 +1,213 @@
+# Computing Experiment Datasets #1: Experiment Subjects
+
+This Lab is part of a multi-part series focused on computing useful experiment datasets. In this Lab, we'll use [PySpark](https://spark.apache.org/docs/latest/api/python/index.html) to compute _experiment subjects_ from [Optimizely Enriched Event](https://docs.developers.optimizely.com/optimizely-data/docs/enriched-events-export) ["Decision"](https://docs.developers.optimizely.com/optimizely-data/docs/enriched-events-data-specification#decisions-2) data.
```

Comment:
What does it mean to `compute` experiment subjects? Is it just to count them up? Whats the benefit of doing this? Perhaps this is covered in part 2 of the series, but it seems like each lab should clearly define the value it provides. Otherwise, it seems like it should just be folded into a larger lab.

Reply:
If it is to 'count' the experiment subjects, then we should consider changing the title from `compute experiment subjects` to `count experiment subjects`

Reply:
Ah, good question.  This lab actually computes a table of subjects for each experiment for which there are decisions present in the intput dataset.  The subjects dataset may be joined with other analytics datasets to compute experiment metrics.  I'll try adding something like this in explicitly.

Reply:
FWIW, #2 in this series works by joining subjects with enriched events to compute observations